<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Foto">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>551</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/sofar/Downloads/Да-Да2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>491</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/sofar/Pictures/Saved Pictures/Коил-Коил .jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>521</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="Line" name="line">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>6</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LeftLine"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Right_Line">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="Set_Left">
           <property name="text">
            <string>Задать Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Set_All">
           <property name="text">
            <string>Задать Left and Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Set_Right">
           <property name="text">
            <string>Задать Right</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="Clear_Left">
           <property name="text">
            <string>Очистить Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Clear_All">
           <property name="text">
            <string>Очистить Left and Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Clear_Right">
           <property name="text">
            <string>Очистить Right</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="Move_Left">
           <property name="text">
            <string>Move_Left_To_Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Swap">
           <property name="text">
            <string>Swap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Move_Right">
           <property name="text">
            <string>Move_Right_To_Left</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="baseSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Сделай свой выбор</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Задать Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Задать Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Задать Left and Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Очистить Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Очистить Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Очистить Left and Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Move Left To Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Move Right to Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Swap</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
           </property>
           <property name="text">
            <string>Ты сделал свой выбор?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
